presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7878,8 +7881,15 @@ public void testQuantifiedComparisonCornerCases(String query) {
         List<QueryTemplate.Parameter> quantifier = new QueryTemplate.Parameter("quantifier").of("ALL", "ANY");
         List<QueryTemplate.Parameter> lhs = new QueryTemplate.Parameter("lhs").of("1", "NULL");
         List<QueryTemplate.Parameter> relation = new QueryTemplate.Parameter("relation").of("=", "!=", "<", ">", "<=", ">=");
-        QueryTemplate query = new QueryTemplate("SELECT %lhs% %relation% %quantifier% (%subquery%)");
-        Stream<Object[]> queries = query.replaceAll(subquery, quantifier, lhs, relation).map((String q) -> new Object[] { q });
+        QueryTemplate queryTemplate = new QueryTemplate("SELECT %lhs% %relation% %quantifier% (%subquery%)");
+        ImmutableList<String> excludedInPredicateQueries = ImmutableList.of(
+                "SELECT NULL != ALL (SELECT * FROM (SELECT 1 WHERE false) as empty_table)",
```

Comment:
Are you working on a fix for these? At a minimum, we should add code to fail queries that have this shape so that they don't produce incorrect results.